src/mmw/js/src/core/views.js

```diff
@@ -321,6 +321,8 @@ var MapView = Marionette.ItemView.extend({
                     leafletLayer = new L.TileLayer(tileUrl, {
                         attribution: layer.attribution || '',
                         maxZoom: layer.maxZoom,
+                        maxNativeZoom: layer.maxNativeZoom || layer.maxZoom,
+                        opacity: layer.opacity ? layer.opacity : 1,
                         zIndex: zIndex
```

Comment:
Consider making this more dynamic so that other Leaflet layer settings can be specified in the layer config and will work without further modifications (as described here: https://github.com/WikiWatershed/model-my-watershed/issues/757#issuecomment-137548508).